src/components/Calculator/ScoreTable.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import Threshold from './Threshold'
 import getCourseScore from '../../utils/getCourseScore'
 import facultiesArray from '../../faculties.json'
+import thresholdsArray from '../../thresholds.json'
 
 const ScoreTable = ({ subjects }) => (
   <div>
     <br />
     <br />
```

Comment:
What about using margin-top on .score-table to handle spacing?
That way you would not require this brs and wrapping in div.

Also it's more flexible to use css for that kind of stuff.